.github/CODEOWNERS

```diff
@@ -477,6 +477,7 @@
 /pkg/internal/rsg/           @cockroachdb/sql-queries
 /pkg/internal/sqlsmith/      @cockroachdb/sql-queries
 /pkg/internal/team/          @cockroachdb/test-eng
+/pkg/internal/workloadreplay/          @cockroachdb/test-eng-noreview
```

Comment:
I think we want `@cockroachdb/test-eng` not `@cockroachdb/test-eng-noreview` here, no?